.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,28 @@
+<!--
+Thank you for reporting an issue.
+
+Please fill in as much of the template below as you're able.
+
+Node Version: output of `node -v`
+Npm Version: output of `npm -v`
+Platform: output of `uname -a` (UNIX), or version and 32 or 64-bit (Windows)
+Step to reproduce :  please provide clear steps to reproduce the bug
+Current behaviour: how the software behave currently
+Expected behaviour : how the software should behave
+Workaround: workaround the issue if there is any (this will help users having the same issue)
+
+If possible, please provide GIF/video that demonstrates the problem, keeping it as
+simple as you are able.
+It would be nice to assign a label to the issue if you find it relevant
+-->
+
+* **Node Version**:
+* **Npm Version**:
```

Comment:
I was thinking maybe users will report issues about the code like "kap is not building for me". I will remove them for now, this can be added later I guess